lib/lita/default_configuration.rb

```diff
@@ -138,6 +138,10 @@ def robot_config
             end
           end
         end
+        default_formatter = proc do |severity, datetime, _progname, msg|
```

Comment:
This can be done inline as it's passed to `config` – check out the call for setting up `error_handler` just below it.